tests/lib/restmail.js

```diff
@@ -7,15 +7,33 @@ define([
   'intern/node_modules/dojo/Promise'
 ], function (request, Promise) {
 
-  function waitForEmail(uri, number) {
+  /**
+   * Wait for an email.
+   *
+   * @param {string} uri - email uri
+   * @param {number} number
+   * @param {object} [options]
+   *   @param {number} [options.minAttemptsBeforeLog] - Minimum number of
+   *   attempts before attempts are logged. Defaults to 2.
+   *   @param {number} [options.maxAttempts] - number of email fetch attempts
+   *   to make. Defaults to Infinity.
```

Comment:
Was there a reason to default to infinity? Maybe reuse 10 maxAttempts from above?